src/functions.ts

```diff
@@ -42,6 +42,13 @@ export const validateAndCreateFunctions = async (
   }
 };
 
+const functionPathHasDefaultExport = async (
+  functionFilePath: string,
+) => {
+  const functionModule = await import(`file://${functionFilePath}`);
```

Comment:
Random thought: we could potentially enforce the type of `default export` returned by userland function code, ensuring that they are e.g. exporting the return of `SlackFunction`.

Not a blocker for this PR but just food for thought / future enhancement.

Reply:
Thinking more about how these default exports are consumed at runtime, and in particular:

- At runtime, [if no `default` export exists, the runtime will throw an exception](https://github.com/slackapi/deno-slack-runtime/blob/main/src/run-function.ts#L22-L26), and
- The [runtime directly invokes the `default` export as a function](https://github.com/slackapi/deno-slack-runtime/blob/main/src/run-function.ts#L37),

Perhaps we should ensure that the `default` export is, at least, a function? Maybe not as far as my previous comment suggested (to check the specific return type), but maybe as simple as a `typeof functionModule.default == 'function'` check?

Reply:
Good call to ensure the default export is a function, like that the footprint is minimal

Reply:
Ive tested and implemented the `typeof functionModule.default == 'function'` check good idea @filmaj 💯